client/src/components/graph.js

```diff
@@ -0,0 +1,229 @@
+import { Component, Fragment } from 'preact'
+import { getGraph } from '../api/scoreboard'
+import withStyles from './jss'
+import { memo } from 'preact/compat'
+
+const height = 400
+const stroke = 5
+const axis = 20
+const axisGap = 10
```

Comment:
We may want to make these props, `height` could be useful to have controllable.

Reply:
An alternative is to have the height be calculated automatically based on the size of the containing element (plotly has an option to do this), but that could become complex to implement.